### ISSUE TITLE
How pass ssl certificate to seleniod in codeception 

### ISSUE THREAD

**Issue by saiedtrife** [Codeception/Codeception#5048]
I have test when I will need to open browser with ssl certificate , the certificate is installed in my local chrome driver.

My situation is I need to pass local chrome configuration to chorme is open with seleniod 

here is my yml file that I use to configure with seleniod: 

```
modules:
    enabled:
        - Setter:
              source: cash
        - WebDriver:
                         browser: chrome
                         url: 'https://creditplus.ru/'
                         host: 192.168.4.117
                         window_size: 1376x768
                         restart: true
                         port: 4444
                         capabilities:
                          browserName: "chrome"
                          version: "65"
                          browserTimeout: 500
                          enableVNC: true
                          sessionTimeout: 120
                          name: "Cash here"
```

I tried to use the chrome profile like it shows below without any success:

```
modules:
    enabled:
        - Setter:
              source: cash
        - WebDriver:
                         browser: chrome
                         url: 'https://creditplus.ru/'
                         host: 192.168.4.117
                         window_size: 1376x768
                         restart: true
                         port: 4444
                         capabilities:
                          browserName: "chrome"
                          version: "65"
                          browserTimeout: 500
                          enableVNC: true
                          sessionTimeout: 120
                          name: "Cash here"
                          chromeOptions:
                           args: "user-data-dir=/home/strefi/.config/google-chrome/Default"]
```

Please if someone have the situation with seleniod and Codeception help me.

In summary:
1- how I can export ssl certificate from local browser to seleniod in Codeception PHP
